src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -48,10 +49,20 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TestTargetFrameworkPath Include="$(TestPath)%(TestTargetFramework.Folder)"/>
+    <TestsSuccessfulSemaphore Include="@(TestTargetFrameworkPath->'%(FullPath)\$(MSBuildProjectFile).tests.lastsucceeded')"/>
```

Comment:
Why isn't the test results file enough for a semaphore? Is it because it also exists when the a failure occurs?